packages/backend/src/services/UserService.ts

```diff
@@ -236,4 +251,51 @@ export class UserService {
         });
         return user;
     }
+
+    async getPasswordResetLink(code: string): Promise<PasswordResetLink> {
```

Comment:
What is the purpose of this? Is this so the frontend can verify the password reset link before the user fills in the password reset form? Does that create a better UX? And what are the security implications?

Reply:
> What is the purpose of this? Is this so the frontend can verify the password reset link before the user fills in the password reset form? Does that create a better UX? 

Correct

>  And what are the security implications?

I can't think of any.